### ISSUE TITLE
How to get candlestick data from websocket/http APIs

### ISSUE THREAD

**Issue by adityamertia** [krystianity/node-bitstamp#22]
Hi,
Is there a way to fetch 1 min candlestick data? If yes how? I do not see it in API docs of bitstamp.
Thanks

**Comment by krystianity** [comment]
I dont think that this is possible with a single API call; you will have to collect the data and build the candle-sticks for yourself. But that should not be too difficult :)

**Comment by sullyfischer** [comment]
I'm also facing this issue. I don't see any endpoint for candle sticks data.
